runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -89,7 +127,7 @@ private static void loadVmProperties() {
             metricsCsvReportingEnabled = metricsCsvInterval > 0 ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1125)